src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesAppDeployer.java

```diff
@@ -253,19 +250,8 @@ private PodSpec createPodSpec(String appId, AppDeploymentRequest request, Intege
 		ImagePullPolicy pullPolicy = deduceImagePullPolicy(properties, request);
 		container.setImagePullPolicy(pullPolicy.name());
 		// Add volumes and mounts
-		if (properties.getHostVolumeMounts() != null) {
-			container.setVolumeMounts(properties.getHostVolumeMounts().stream()
-					.map(hvm -> new VolumeMount(hvm.getContainerPath(), hvm.getName(), hvm.isReadOnly(), null))
-					.collect(Collectors.toList()));
-			podSpec.withVolumes(properties.getHostVolumeMounts().stream()
-					.map(mv -> {
-						Volume volume = new Volume();
-						HostPathVolumeSource hostPath = new HostPathVolumeSource();
-						hostPath.setPath(mv.getHostPath());
-						volume.setHostPath(hostPath);
-						volume.setName(mv.getName());
-						return volume;
-					}).collect(Collectors.toList()));
+		if (properties.getVolumeMounts() != null) {
```

Comment:
Isn't it a bit limiting to require volume mounts to be in the deployer configuration? If I wanted to use a `hostPath` volume I would have to re-deploy the server after re-configuring it?